### ISSUE TITLE
Why is the owl carousel creating a white space/gap? 

### ISSUE THREAD

**Issue by jaxboy** [OwlCarousel2/OwlCarousel2#1338]
Right now the carousel image is expanded full width and is responsive, but it's too big. How do I adjust the size to make it smaller? I am using owl carousel. There is an awkward white gap after adding in the carousel, on the right size. The horizontal scrollbar appears where as before without the carousel, there was no horizontal scrollbar and the width of the image expanded full width without having to use the horizontal scrollbar.

What is wrong with my code? Sorry I can't upload the site because I was not giving permission to access to the companies server yet. I am just building it locally for now.

http://codepen.io/xaznbabyx/pen/grKGQd

**Comment by dallanlee** [comment]
Your CodePen link is not showing me anything. You don't have absolute URLs to the images so it's all blank where there should be images. And you pasted so much crap in the JS panel that I don't think anyone wants to search for your OwlCarousel call. (I think you're just using the defaults, aren't you?) I don't mean to be rude just to be rude but have you written even 10% of the markup or code? It looks like you've "borrowed" almost everything. Have you even read through the documentation and options for OwlCarousel? [Read about what margins are](https://owlcarousel2.github.io/OwlCarousel2/docs/api-options.html#margin). That would be my first guess. I just get frustrated with people who fake it in this industry. I'm all for learning. We all start at the beginning. Anyway, I know you didn't ask but I also just want to say, you could work on a staging server while not having access to a live server. You can't really know exactly what your code is gonna do if you're only working locally anyway.

**Comment by jaxboy** [comment]
I'm not trying to fake it in the industry. I am not working at an actual company and I am all for learning too.  The person I am helping out is going to provide a CTO to mentor me in the process, but she wants me to help her with the website and other graphic design. I admit that most of these codes are defaults because I don't know javascript too well. Javascript isn't the easiest thing for me. I tried learning it, I don't quite understand it, not saying that I won't try to learn it. This my very first time I am doing something that's not a school project. So I am still learning and I'm still a beginner. Never said I was a professional and faking it.... mind you. I mostly borrowed the javascript. I copied the Jquery.min.js in there, then I copied the owl.carousel.js and part of the css in there. The rest was from my class. 

The person whom I was supposed to ask for help isn't available to help me yet. I'm supposed to use heroku to upload it on the server. It's ok, I'll figure it out. Thanks anyways.

**Comment by daviddeutsch** [comment]
@dallanlee That sort of rudeness is completely uncalled for and, worse than that, unproductive. Congratulations, you made a person starting out feel terrible. Please do not do that.

@jaxboy The problem with your example was that it did not isolate the carousel, making it hard to figure out what is going on. Funnily enough, isolating where your problem is often has the benefit that you find your own mistakes quicker. Perhaps try that first and let me know whether you still have issues.

I have a hunch that the solution to your issue is either tweaking the `margin` parameter or modifying your CSS to not add a margin.

**Comment by dallanlee** [comment]
@daviddeutsch You're right. I don't know why I acted that way. I must have been letting off frustration from the day. No excuses, though. I admit that was uncalled for. @jaxboy I apologize.

**Comment by daviddeutsch** [comment]
Thank you for being considerate. We're in an industry where you never stop learning and . Seeing people be mean about a lack of knowledge is a

**Comment by dallanlee** [comment]
@jaxboy I've forked your Pen and added the assets via CodePen. (If you look at settings in CodePen, there are CSS and JS tabs to load in things like Bootstrap and jQuery). [See my forked Pen here](http://codepen.io/dallanlee/pen/MyXqWO). I added the OwlCarousel2 minified CSS in the CSS panel but your CSS is still just below it. And I added the minified OwlCarousel2 JS in the JS panel and then call the plugin on the element below the code. (Note that I'm using the latest v2.1.1 release of OwlCarousel2) I don't see a margin though..

**Comment by daviddeutsch** [comment]
We're in an industry where you never stop learning (which can be super scary) and nobody can claim to know it all. Seeing people be mean about a lack of knowledge irks me a lot because I deal with plenty of people who have had a slant in their progress because of a rude experience. I'm not a fan of completely sanitized conversations, either, we only need a basic level of common decency and assuming something productive about where the other person is coming from. 

Thank you for being considerate. And thanks for going the extra mile. I hope we haven't lost @jaxboy completely.

**Comment by jaxboy** [comment]
@dallanlee It's ok. We're all still learning here. I'm still seeing the white margin on the right side. How did you download the new version of Carousel v2.1.1?? 

Anyways omg! my problem was solved and it was because I made a silly typo that someone else had pointed out! I mispelled container... smh... This is why coding fries my brain cells! But thanks for trying to help me. And also thank you David for being so kind. Now I'm trying to figure out how to move the left and right arrow navs and center it. I feel reliefed now...  !_!

<section id="app" class="section-padding">
	
        <div class="container">
